cassandra/cqlengine/statements.py

```diff
@@ -550,6 +550,7 @@ class SelectStatement(BaseCQLStatement):
     def __init__(self,
                  table,
                  fields=None,
+                 distinct_fields=[],
```

Comment:
Would it be okay to default to None? Looks safe here, but mutable defaults are usually discouraged.